lib/signs_ui/signs/sign.ex

```diff
@@ -105,6 +105,48 @@ defmodule SignsUI.Signs.Sign do
     }
   end
 
+  @spec from_config(any(), map()) :: t()
```

Comment:
Shouldn't this be `id()` rather than `any()` for the type of `sign_id`?

Reply:
Well, it's data the JS is sending from the frontend, but it can be narrowed down to `String.t()` at least, so I'll do that.